welltestpy/process/processlib.py

```diff
@@ -240,3 +246,47 @@ def filterdrawdown(observation, tout=None, dxscale=2):
         hout = np.interp(tout, time, head)
 
     return observation(time=tout, observation=hout)
+
+
+def smoothing_derivative(observation, bourdet=True):
```

Comment:
Please use a keyword `method="bourdet"` with default value "bourdet".
Then check for the method and raise an error if it is unknown.